ui/src/app/connect/components/adapter-configuration/start-adapter-configuration/adapter-options-panel/adapter-options-panel.component.ts

```diff
@@ -36,6 +36,9 @@ export class SpAdapterOptionsPanelComponent {
     @Input()
     dataCy: string;
 
+    @Input()
+    isChecked = false;
```

Comment:
What is the default behaviour? Adapters are started upon creation?

Reply:
Ok, I see that `startAdapterNow` is true. Why is `isChecked` initially set to false?

Reply:
The default value is `false` only for the `AdapterOptionsPanel` component. This component is also used for other configurations like **Remove Duplicate**. Since the default value for all other configurations is `false`, I decided to set the default value of this component to `false` as well.